src/strands/agent/agent.py

```diff
@@ -546,7 +546,7 @@ def _execute_event_loop_cycle(self, callback_handler: Callable[..., Any], kwargs
 
         try:
             # Execute the main event loop cycle
-            stop_reason, message, metrics, state = event_loop_cycle(
+            events = event_loop_cycle(
```

Comment:
Out of curiosity, what's keeping us needing to pass in `callback_handler` to the `event_loop_cycle`?

Reply:
If we're releasing today - we might want to try to get that warning in before we release.  That would give folks at least a week or two before we yank it out